public/layout.css

```diff
@@ -25,6 +25,12 @@ html, body, uiview {
 	margin: 0;
 }
 
+@media (min-width: 600px) {
+	.container {
```

Comment:
Is this necessary?  This seems weird – if we wanted this behavior, it seems like it would be better to change the style up above to

```css
@media (min-width: 600px) {
	:root{
		--containerPadding: 64px;
	}
}
```

...but I'm not sure we really need that much padding at that width.

Reply:
🐳 

Still relevant I think 😅

Reply:
hah, depending on how you read my comments, this comment is unclear about what it's commenting on... but it's meant to be threaded with https://github.com/TehShrike/memorize-text-app/pull/17#discussion_r255172863